packages/tenant-process/test/revokeVerifiedAttribute.test.ts

```diff
@@ -0,0 +1,322 @@
+    await writeInReadmodel(
+      toReadModelAgreement(agreementEservice1),
+      agreements
+    );
+    expect(
+      tenantService.revokeVerifiedAttribute(
+        {
+          tenantId: targetTenant.id,
+          attributeId: verifiedAttribute.id,
+          revokerId: revokerTenant.id,
+          correlationId: generateId(),
+        },
+        genericLogger
+      )
+    ).rejects.toThrowError(tenantNotFound(targetTenant.id));
+  });
+  it("Should throw attributeNotFound if the attribute doesn't exist", async () => {
+    const tenantWithoutSameAttributeId: Tenant = {
+      ...targetTenant,
+      attributes: [
+        {
+          ...verifiedAttribute,
+          id: generateId(),
+          verifiedBy: [
+            {
+              ...getMockVerifiedBy(),
+              id: revokerTenant.id,
+            },
+          ],
+          revokedBy: [{ ...getMockRevokedBy() }],
+        },
+      ],
+    };
+
+    await addOneTenant(tenantWithoutSameAttributeId);
+    await addOneTenant(revokerTenant);
+    await writeInReadmodel(toReadModelEService(eService1), eservices);
+    await writeInReadmodel(
+      toReadModelAgreement(agreementEservice1),
+      agreements
+    );
+    expect(
+      tenantService.revokeVerifiedAttribute(
+        {
+          tenantId: tenantWithoutSameAttributeId.id,
+          attributeId: verifiedAttribute.id,
+          revokerId: revokerTenant.id,
+          correlationId: generateId(),
+        },
+        genericLogger
+      )
+    ).rejects.toThrowError(attributeNotFound(verifiedAttribute.id));
+  });
+  it("Should throw attributeRevocationNotAllowed if the organization is not allowed to revoke the attribute", async () => {
+    const tenantWithVerifiedAttribute: Tenant = {
+      ...targetTenant,
+      attributes: [
+        {
+          ...verifiedAttribute,
+          verifiedBy: [
+            {
+              ...getMockVerifiedBy(),
+              id: generateId(),
+            },
+          ],
+          revokedBy: [{ ...getMockRevokedBy() }],
+        },
+      ],
+    };
+
+    await addOneTenant(tenantWithVerifiedAttribute);
+    await addOneTenant(revokerTenant);
+    await writeInReadmodel(toReadModelEService(eService1), eservices);
+    await writeInReadmodel(
+      toReadModelAgreement(agreementEservice1),
+      agreements
+    );
+
+    expect(
+      tenantService.revokeVerifiedAttribute(
+        {
+          tenantId: tenantWithVerifiedAttribute.id,
+          attributeId: verifiedAttribute.id,
+          revokerId: revokerTenant.id,
+          correlationId: generateId(),
+        },
+        genericLogger
+      )
+    ).rejects.toThrowError(
+      attributeRevocationNotAllowed(targetTenant.id, verifiedAttribute.id)
+    );
+  });
+  it("Should throw verifiedAttributeSelfRevocationNotAllowed if the organizations are not allowed to revoke own attributes", async () => {
+    await addOneTenant(targetTenant);
```

Comment:
```suggestion
  it("Should throw verifiedAttributeSelfRevocationNotAllowed when trying to revoke own attributes", async () => {
```
`targetTenant` shouldn't be needed

Reply:
This test still has to be renamed